eq-author/src/App/QuestionnairesPage/index.js

```diff
@@ -75,30 +78,31 @@ export class UnconnectedQuestionnairesPage extends React.PureComponent {
 
   render() {
     const { onCreateQuestionnaire } = this.props;
-
     return (
       <Titled title={this.renderTitle}>
         <BaseLayout title={"Your Questionnaires"}>
-          <MainCanvas>
-            <StyledButtonGroup horizontal>
-              <Button
-                onClick={this.handleModalOpen}
-                primary
-                data-test="create-questionnaire"
-              >
-                Create
-              </Button>
-              <QuestionnaireSettingsModal
-                isOpen={this.state.isModalOpen}
-                onClose={this.handleModalClose}
-                onSubmit={onCreateQuestionnaire}
-                confirmText="Create"
-              />
-            </StyledButtonGroup>
-            <StyledCenteredPanel>
-              <Query query={QUESTIONNAIRES_QUERY}>{this.renderResults}</Query>
-            </StyledCenteredPanel>
-          </MainCanvas>
+          <ScrollPane permanentScrollBar>
+            <MainCanvas width="70em">
```

Comment:
Remove this for now as the table doesn't have extra columns yet so doesn't need to be this wide.

Reply:
Done. Set to default of 55em.